python_ta/cfg/graph.py

```diff
@@ -1,6 +1,11 @@
 from __future__ import annotations
 
-from typing import Any, Generator, List, Optional, Set, Tuple, Union
+from typing import Any, Dict, Generator, List, Optional, Set
+
+try:
+    from z3 import ExprRef
+except ImportError:
+    ExprRef = None
```

Comment:
Great, you can set this to `Any` instead

Reply:
Do you mean that I should change the type annotation for `_z3_vars` to `_z3_vars: Dict[str, Any]` instead of `_z3_vars: Dict[str, ExprRef]`?

Reply:
Oh, I didn't realize Any could be a value and not just a type annotation... thanks!